### ISSUE TITLE
create-release failed when release contains symlink directories

### ISSUE THREAD

**Issue by qibobo** [cloudfoundry/bosh-cli#476]
Noticed that in release 5.2.1 there is a fix:
Fix issue when creating releases from within symlink directories

But I still encounter the symlink error:

**scalerui is soft link directory.**
```
qiyang@qiyangdeMacBook-Pro scalerui-release (develop) $ bosh5 create-release --force
Building a release from directory '/Users/qiyang/dev/openjavaspace/app-autoscaler-bluemix/releases/app-autoscaler-ui/scalerui-release':
  - Constructing packages from directory:
      - Reading package from '/Users/qiyang/dev/openjavaspace/app-autoscaler-bluemix/releases/app-autoscaler-ui/scalerui-release/packages/scalerui':
          Collecting package files:
            Missing files for pattern 'scalerui/**/*'
  - Constructing jobs from manifest:
      - Expected to find package 'nodejs' since it's a dependency of job 'scalerui'

Exit code 1
qiyang@qiyangdeMacBook-Pro scalerui-release (develop) $ bosh5 -version
version 5.2.1-24101936-2018-08-27T21:54:39Z

Succeeded
```

**Comment by belinda-liu** [comment]
Hi @qibobo,

The fix in 5.2.1 regarding symlinks doesn't change our original intended behavior, which was updated in [5.0.1](https://github.com/cloudfoundry/bosh-cli/releases/tag/v5.0.1). Any symlink in a release directory will not be resolved, and instead tracked literally. We changed the behavior in 5.0.1 to avoid resolving symlinks during `create-release`.

In this release, are you able to avoid `scalerui` being a simlink? Perhaps you can instead use submodules or directly move those files to live in that directory.

Hope this helps,
@belinda-liu && @dpb587, CF BOSH Team

**Comment by pivotal-mp** [comment]
Closing this issue for now.
